aws/v9.3.9/README.md

```diff
@@ -0,0 +1,23 @@
+# :zap: Giant Swarm Release v9.3.9 for AWS :zap:
+
+**If you are upgrading from 9.3.8, upgrading to this release will not roll your nodes.**
+
+
+This is a patch release to update cluster-operator mitigating the unexpected deletion of app CRs in clusters. 
```

Comment:
```suggestion
This patch release fixes the problem causing the accidental deletion and reinstallation of Preinstalled Apps (such as CoreDNS) in 9.x.x tenant clusters.

Please upgrade all older clusters to this version in order to prevent downtime. 
```

Reply:
@tomahawk28 what will they prevent from upgrading to this release? Is it downtime? I am not sure of that part.

Reply:
The bug would delete coreDNS and other apps so  it would bring downtime to the tenant clusters. is that what you are asking? The change looks good so I'm going to commit your suggestion.